packages/core/src/codec/clusterProxy.ts

```diff
@@ -16,7 +16,7 @@ export interface RawClusterProxyArgs {
 export const ClusterProxyArgs = createBytesCodec({
   pack(packable: PackableClusterProxyArgs): Uint8Array {
     const id = blockchain.Byte32.pack(packable.id);
-    const minPayment = Uint8Opt.pack(packable.minPayment);
+    const minPayment = Uint64Opt.pack(packable.minPayment);
```

Comment:
Making direct changes to the pack/unpack method of a molecule seems inappropriate because the spore-sdk should support both new and old versions of the spore-contracts, instead of only the latest version. For reference on how ClusterDataV1 and ClusterDataV2 were handled, please check here: https://github.com/sporeprotocol/spore-sdk/blob/beta/packages/core/src/codec/cluster.ts#L69-L97.